### PR TITLE
[CI] Run tests after months not running them.

### PR DESCRIPTION
**Pull request by mandel-macaque**
This will probably 💣 💥

### CODE REVIEW

**Review by rolfbjarne — APPROVED** [review]

File: devops/automation/run-tests.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash -e
 
+echo $SOM_PATH
+unset SOM_PATH
```
Comment (tj-devel709) [inline devops/automation/run-tests.sh:4]: What is this doing?

**Comment by tj-devel709** [comment]
Also I remember that when I started, I did a little work with device tests and a lot of them would run locally but they would not run in vsts because of naming conflicts. Just wanted to make sure this is not the issue that is happening now here. @mandel-macaque @stephen-hawley

File: devops/automation/run-tests.sh (hunk 1 above)
Comment (mandel-macaque) [reply devops/automation/run-tests.sh:4]: I think it was stuff left behind.

**Comment by mandel-macaque** [comment]
@tj-devel709 this issue I was fixing is that a commit you made in Nov disabled the tests on CI and have  been ran since then :/

Now we have failing tests that we did not know about.

**Review by stephen-hawley — APPROVED** [review]

**Pull request merged**